apps/tangle-dapp/constants/evmActions.ts

```diff
@@ -32,6 +31,8 @@ export const PAYMENT_DESTINATION_OPTIONS = [
 ];
 
 const tangleTestnetConfig = chainsConfig[PresetTypedChainId.TangleTestnetEVM];
+
+// TODO: Explain why this is being done.
 delete tangleTestnetConfig.contracts;
```

Comment:
git blame, who put this there?

Reply:
@devpavan04, I'm curious about this delete operation; do you happen to know what its purpose is?